solr/vufind/biblio/conf/schema.xml

```diff
@@ -232,6 +244,7 @@
    <dynamicField name="*_boolean" type="boolean" indexed="true" stored="true"/>
    <dynamicField name="*_geo" type="geo" indexed="true" stored="true" multiValued="false" />
    <dynamicField name="*_geo_mv" type="geo" indexed="true" stored="true" multiValued="true" />
+   <dynamicField name="*_autocomplete" type="text_autocomplete" indexed="true" stored="true" multiValued="true"/>
```

Comment:
Would it be more clear to add _mv on the end of this suffix for consistency with other multi-valued suffixes? Or should this only be used with single-valued fields, in which case we should set multiValued to false here.

It seems like this approach might lead to strange results with multi-valued fields, unless I am missing something, because if I have two values, A and B, and my search query matches A, won't the recommendation list include both A *and* B?

Reply:
Yes, you are right, it works best for single-value fields. I will fix it in schema.xml.